src/webview/memory-webview-view.tsx

```diff
@@ -107,6 +112,37 @@ class App extends React.Component<{}, MemoryAppState> {
         }
     }
 
+    protected handleWrittenMemory(writtenMemory: WrittenMemory): void {
+        if (!this.state.memory) {
+            return;
+        }
+        if (this.state.memoryReference === writtenMemory.memoryReference) {
+            // catch simple case
+            this.refreshMemory();
+            return;
+        }
+        try {
+            // if we are dealing with numeric addresses (and not expressions) then we can determine the overlap
+            const written: BigIntMemoryRange = {
+                startAddress: BigInt(writtenMemory.memoryReference),
+                endAddress: BigInt(writtenMemory.memoryReference) + BigInt(writtenMemory.count ?? 0)
+            };
+            const shown: BigIntMemoryRange = {
+                startAddress: this.state.memory.address,
+                endAddress: this.state.memory.address + BigInt(this.state.memory.bytes.length)
```

Comment:
Good question! I think in theory you might be right but in this case we only do an overlap calculation (bigint arithmetic) and we also take the bytes of the current memory and not actually the number of addresses shown. So I believe we are fine even though we have to go with the `endAddress` terminology as it is part of the `BigIntMemoryRange` interface. What do you think? I'll add a comment to avoid confusion.